src/unix/udp.c

```diff
@@ -167,13 +217,24 @@ static void uv__udp_recvmsg(uv_udp_t* handle) {
 
   do {
     buf = uv_buf_init(NULL, 0);
-    handle->alloc_cb((uv_handle_t*) handle, 64 * 1024, &buf);
+    handle->alloc_cb((uv_handle_t*) handle, UDP_DGRAM_MAXSIZE, &buf);
     if (buf.base == NULL || buf.len == 0) {
       handle->recv_cb(handle, UV_ENOBUFS, &buf, NULL, 0);
       return;
     }
     assert(buf.base != NULL);
 
+    /* TODO: test of recvmmsg support just once */
+    uv__recvmmsg(handle->io_watcher.fd, NULL, 0, 0, NULL);
```

Comment:
Could you prefer using an explicit return value over `errno` when possible.

Reply:
Fixed in `uv__udp_mmsg_init()`